dev-board/Src/main.cpp

```diff
@@ -139,6 +140,9 @@ int main(void)
   bms0.SendData();
   /* USER CODE END 2 */
 
+  //IMU INIT

```

Comment:
Can you follow this comment style https://github.com/Solar-Gators/Telemetry_2019-2020/pull/4/files#diff-597c7667fd484ee38c29c407028a15aaR125